Odata-docs/extensions/migration.md

```diff
@@ -0,0 +1,262 @@
+﻿# Introduction
```

Comment:
Please change the document title from Introduction to something relevant to the topic being documented.

Reply:
The document does not exist in the table of contents. Please add the to the toc.yml file. I would suggest adding a "Migration" node under the Legacy API node.

Reply:
Update all but the first H1 to H2 heading through the entire document.

Reply:
document needs a AppliesTo tag. Please refer to the following page to see an example of the appliesto tag.
https://review.docs.microsoft.com/en-us/odata/webapi/getting-started?branch=pr-en-us-73

Reply:
changed